covalent/_data_store/datastore.py

```diff
@@ -34,15 +34,15 @@
 class DataStore:
     def __init__(
         self,
-        db_URL,
+        db_URL: Optional[str] = None,
         storage_backend_map: Dict[str, StorageBackend] = default_backend_map,
         initialize_db: bool = False,
         **kwargs,
     ):
         if db_URL:
             self.db_URL = db_URL
         else:
-            self.db_URL = "sqlite+pysqlite://" + get_config("workflow_data.db_path")
+            self.db_URL = "sqlite+pysqlite:///" + get_config("workflow_data.db_path")
```

Comment:
@scottwn @FyzHsn  It seemed that there was a missing slash in order to use an absolute path like the one specified by the config `workflow_data` (Refer to https://docs.sqlalchemy.org/en/14/core/engines.html#sqlite). Was this working for y'all before?

Reply:
No sir! It wasn't working. So far, we were manually inserting the DB URL. Thanks for adding the fix.

Reply:
Got it, thanks !